db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
```

Comment:
again, think we can remove the commented out lines

Reply:
Deleted